test/benchmark/sqlite/ext2_deletes_between/config.json

```diff
@@ -0,0 +1,20 @@
+{
+    "alert_threshold": "125%",
+    "alert_tool": "customSmallerIsBetter",
+    "search_pattern": "10000 DELETEs, numeric BETWEEN, indexed....",
+    "result_index": "8",
+    "result_substring": "1,5",
+    "description": "sqlite-speed-test: 10000 DELETEs, numeric BETWEEN, indexed",
+    "title": "[Ext2] The deletes-between result of sqlite speedtest",
+    "linked_benchmarks": [
```

Comment:
This is a poor API design to achieve the goal of collecting multiple benchmark results by running a single benchmark program.

First of all, the `run.sh` script is duplicated across the so-called "linked" benchmark test directories. Remember the DRY principle. Code duplication should be avoided.

Second, the code organization does not reflect the fact that these `sqlite/ext2_xxx` benchmark tests actually are "linked". One has no way to know which benchmarks are "linked" unless she opens the `config.json` file under every benchmark directory and check if it has a `linked_benchmarks` field.

Third, every benchmark among a set of linked benchmarks is equally important. So the natural mental model is a _set_, not a _list_. Thus, people do not expect one of the linked benchmark to be the _head_ of the list. There shouldn't be a head at all.

People do not expect this set to have a link